checks.d/zk.py

```diff
@@ -96,6 +96,20 @@ def check(self, instance):
                 self.service_check('zookeeper.mode', status, message=message,
                                    tags=sc_tags)
 
+        # Read ouput from `mntr` output.
+        try:
+            mntr_out = self._send_command('mntr', *cx_args)
+        except ZKConnectionFailure:
+            self.increment('zookeeper.timeouts')
+            raise
+        else:
+            # Parse the response
+            metric_hash, state = self.parse_mntr(mntr_out)
+
+            # Write the data
+            for name in metric_hash:
+                self.gauge(name, metric_hash[name], tags=tags + [state])
```

Comment:
They need the same format as the others. `zookeeper.<metric name>`, a simple concat should do it.